api/services/EmailService.js

```diff
@@ -46,6 +47,9 @@ function send(options, tpl, context) {
     message: options,
     locals: context,
   };
+  logger.info(
+    `[EmailService->send] About to send ${tpl} email to ${options.to}`,
+  );
```

Comment:
`logger.debug` for this one?

Reply:
logger.debug would mean that it wouldn't be logged in production, and I think it might be interesting to have some logs of the emails that are sent out no ?

Reply:
I didn't think of that. carry on!